e2e/rbd.go

```diff
@@ -1984,6 +1986,115 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a encrypted block pvc and verify the image size", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					framework.Failf("failed to delete storageclass: %v", err)
+				}
+				err = createRBDStorageClass(
+					f.ClientSet,
+					f,
+					defaultSCName,
+					nil,
+					map[string]string{"encrypted": "true", "encryptionType": util.EncryptionTypeBlock.String()},
+					deletePolicy)
+				if err != nil {
+					framework.Failf("failed to create storageclass: %v", err)
+				}
+
+				var (
+					imageSize       uint64
+					resizeImageSize uint64
+					sizeInBytes     int64
+				)
+
+				//nolint:goconst // The string "1Gi" is used multiple times in rbd.go, so it's not a const value.
+				pvcSize := "1Gi"
+				if sizeInBytes, err = helpers.RoundUpToB(resource.MustParse(pvcSize)); err != nil {
+					framework.Failf("failed to parse pvc size: %v", err)
+				}
+				imageSize = uint64(sizeInBytes) + util.GetLuksHeaderSize()
+
+				resizePVCSize := "2Gi"
+				if sizeInBytes, err = helpers.RoundUpToB(resource.MustParse(resizePVCSize)); err != nil {
+					framework.Failf("failed to parse resize pvc size: %v", err)
+				}
+				resizeImageSize = uint64(sizeInBytes) + util.GetLuksHeaderSize()
+
+				pvc, err := loadPVC(rawPvcPath)
+				if err != nil {
+					framework.Failf("failed to load PVC: %v", err)
+				}
+				pvc.Namespace = f.UniqueName
+				pvc.Spec.Resources.Requests[v1.ResourceStorage] = resource.MustParse(pvcSize)
+
+				app, err := loadApp(rawAppPath)
+				if err != nil {
+					framework.Failf("failed to load application: %v", err)
+				}
+				app.Labels = map[string]string{"app": "rbd-pod-block-encrypted"}
```

Comment:
```go
labelKey:="app"
labelVal:= "rbd-pod-block-encrypted"
app.Labels = map[string]string{labelKey:labelVal}

opt := metav1.ListOptions{
	LabelSelector: fmt.Sprintf("%s=%s",labelKey,labelVal)
}
```
using like above is less error prone, if we check it in one place we don't need to change it in other place